src/components/common/ColonyMembers/MembersFilter/MembersFilter.tsx

```diff
@@ -66,9 +66,11 @@ const MembersFilter = ({ handleFiltersCallback, isRoot }: Props) => {
                 </span>
                 {showReset && (
                   <Button
+                    disabled={!isValid || isSubmitting}
                     text={MSG.reset}
+                    type="submit"
                     appearance={{ theme: 'blue' }}
-                    onClick={() => resetForm()}
+                    onClick={() => reset()}
```

Comment:
Could be `onClick={reset}`

Reply:
@willm30 After making this change the reset stops working, also Im getting this error, 
`Type 'UseFormReset<{ memberType: MemberType; verificationType: VerificationType; }>' is not assignable to type 'MouseEventHandler<HTMLButtonElement>'.
  Types of parameters 'values' and 'event' are incompatible.
    Type 'MouseEvent<HTMLButtonElement, MouseEvent>' is not assignable to type '{ memberType: MemberType; verificationType: VerificationType; } | { memberType?: MemberType | undefined; verificationType?: VerificationType | undefined; } | ResetAction<...> | undefined'.`

Reply:
Ah yes, fair enough